### PR TITLE
changelog: Clever Tools 3.2.0

### PR DESCRIPTION
**Pull request by davlgd**
## Checklist

- [ ] My PR is related to an opened issue : #

**⚠️ If changes affect GitHub Action files**

- [ ] I have added tests to cover my changes : [link](url)

## Pull request type

Please check the type of change your PR introduces:

- [x] Documentation content changes
- [ ] Bugfix on the site
- [ ] Build related changes
- [ ] Other (please describe):
      
## Description

Add Clever Tools 3.2.0 release to changelog

### CODE REVIEW

File: content/changelog/2024-02-07-clever-tools-3.2.0.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    link: https://github.com/hsablonniere
+    image: https://github.com/hsablonniere.png?size=40
+description: And more is coming!
+excludeSearch: true
+---
+
+As mentionned [in the previous release](/changelog/2024-01-25-clever-tools-3.1.0/), the packaging process moved to GitHub Actions to provide a better process, quicker tests and more frequent releases. We start with [the 3.2.0](https://github.com/CleverCloud/clever-tools/releases/tag/3.2.0) adding the log auto retry feature on network failures and some fixes:
+
+* **api:** improve error message with `EAI_AGAIN` and `ECONNRESET` ([b134213](https://github.com/CleverCloud/clever-tools/commit/b134213f30d46dd7f5690a38425deb4fd752148c))
+* delete beta/Jenkins mentions in README.md ([cb32b3b](https://github.com/CleverCloud/clever-tools/commit/cb32b3b0cdaecac5d6198f295eef7c350161ac03))
```
Comment (hsablonniere) [inline content/changelog/2024-02-07-clever-tools-3.2.0.md:21]: suggestion: we can remove this I think

**Review by hsablonniere — APPROVED** [review]
LGTM (small suggestion) :+1:

**Review by cnivolle — DISMISSED** [review]
Looks good to me, I'm aligned with @hsablonniere comment.

File: content/changelog/2024-02-07-clever-tools-3.2.0.md (hunk 1 above)
Comment (cnivolle) [reply content/changelog/2024-02-07-clever-tools-3.2.0.md:21]: Hi @davlgd 👋🏻 
Do you want to make this edit ?

**Review by cnivolle — APPROVED** [review]

**Pull request merged**